### PR TITLE
Fixed a bug: Pi output in tensorflow is incorrect (missing softmax la…

### PR DESCRIPTION
**Pull request by josey4869**
In othello.tensorflow.OthelloNNet.py:

The pi output of the model is

self.pi = Dense(s_fc2, self.action_size) (line 36)

which is incorrect. In fact, it should be

self.pi = tf.nn.softmax(Dense(s_fc2, self.action_size)).

By contrast, the implementation in Keras is correct. In othello.keras.OthelloNNet.py:

self.pi = Dense(self.action_size, activation='softmax', name='pi')(s_fc2) (line 28).

### CODE REVIEW

**Comment by rlronan** [comment]
Softmax crossentropy expects logits in tensorflow, so softmax should not be applied to pi directly.
https://www.tensorflow.org/api_docs/python/tf/compat/v1/losses/softmax_cross_entropy
